cmd/kops/update_cluster.go

```diff
@@ -290,19 +299,29 @@ func RunUpdateCluster(ctx context.Context, f *util.Factory, out io.Writer, c *Up
 		return nil, err
 	}
 
+	minControlPlaneRunningVersion := cluster.Spec.KubernetesVersion
+	if !c.IgnoreVersionSkew {
+		minControlPlaneRunningVersion, err = checkControlPlaneRunningVersion(ctx, cluster.ObjectMeta.Name, minControlPlaneRunningVersion)
+		if err != nil {
+			klog.Warningf("error checking control plane running verion: %v", err)
+		} else {
+			klog.Warningf("successfully checked control plane running version: %v", minControlPlaneRunningVersion)
```

Comment:
Nit: any reason to make this a warning (as opposed to Infof)?

Reply:
oh no... that was just for my testing i fully intended to make it info or debug and then forgot :)